logstash-core/src/test/java/org/logstash/plugins/PluginValidatorTest.java

```diff
@@ -43,6 +43,7 @@ public void testValidOutputPlugin() {
         Assert.assertTrue(PluginValidator.validatePlugin(PluginLookup.PluginType.OUTPUT, Stdout.class));
     }
 
```

Comment:
You are missing the `import org.junit.Ignore;` statement, which is causing the build to fail

Reply:
thanks for heads up, fixed :)